data-structures/trie/trie.py

```diff
@@ -0,0 +1,40 @@
+class trienode:
+	def __init__(self,character):
+		self.character = character
+		self.nextnodes = {}
+		self.isEnd = False
+		self.dataNode = None
+	
+class trie: # use case: username/password
+	def __init__(self):
+		self.start = trienode('')
+	def insert(self,id,data): # returns true on successful insertion (data == password)
```

Comment:
Please use `key` instead of `data` as a variable name.

Reply:
Updated the variable name